actions/workflows/ngi_uu_workflow.yaml

```diff
@@ -284,17 +308,77 @@ workflows:
                     destination_user: "funk_901"
                     destination_path: <% $.irma_remote_path %>
                     rsync_include_file: /etc/arteria/misc/hiseq.rsync
-                    md5_output_file: "ngi.md5"
+                    md5_output_file: "checksums.md5"
                 on-success:
-                  - mark_as_finished
-                  - notify_finished
+                  - irma_check_md5sums
 
-            # TODO Need to add checksum verification on irma
-            # TODO Need to add starting the ngi pipeline on irma
+            irma_check_md5sums:
+               action: core.http
+               input:
+                    method: "POST"
+                    url: <% $.irma_checksum_base_url %>/start/<% $.runfolder_name %>?apikey=<% $.irma_api_key %>
+                    body: '{"path_to_md5_sum_file": "MD5/checksums.md5"}'
+                    verify_ssl_cert: False
```

Comment:
Not at the moment, but I think it would be a good idea to do in the future. I guess that is something for @johanherman to look into with `irma`?